### PR TITLE
Sort answers by score

### PR DESCRIPTION
**Pull request by vermiculus**

### CODE REVIEW

File: sx-question-print.el [hunk 1]
```diff
@@ -179,7 +179,12 @@ QUESTION must be a data structure returned by `json-read'."
   ;; Print everything
   (sx-question-mode--print-section question)
   (sx-assoc-let question
-    (mapc #'sx-question-mode--print-section .answers))
+    (mapc #'sx-question-mode--print-section
+          (cl-sort .answers
```
Comment (Bruce-Connor) [inline sx-question-print.el]: Why not `sort`?

Comment (vermiculus) [inline sx-question-print.el]: I looked at the docs for `sort` (well, the `eldoc`) and it said the sequence had to be a list. I made this patch very on-the-fly this morning and I was running late for work, so I didn't try an alternative XD

Does `sort` work for vectors? That's what `.answers` is.

File: sx-question-print.el [hunk 2]
```diff
@@ -179,7 +179,12 @@ QUESTION must be a data structure returned by `json-read'."
...
+                   ;; Highest-voted first. @TODO: custom sorting
```
Comment (vermiculus) [inline sx-question-print.el]: Notes as #226

File: sx-question-print.el (hunk 1 above)
Comment (Bruce-Connor) [inline sx-question-print.el]: You're right, it seems vector `sort`ing was implemented only recently.

**Pull request merged**